src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/carts.yml

```diff
@@ -0,0 +1,36 @@
+admin_carts_index:
+  path: /
+  methods: [GET]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Order/Cart:index
+    _legacy_controller: AdminCarts
+
+admin_carts_search:
+  path: /
+  methods: [POST]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Order/Cart:search
+    _legacy_controller: AdminCarts
```

Comment:
Will you add legacy_links later ?

Reply:
Yes, after both PRs are merged I'll do another to enable page and delete old files.